src/pages/terms.md

```diff
@@ -1,120 +1,27 @@
 ---
-title: 'Terms and Conditions'
+title: 'Terms of Service'
 layout: '~/layouts/MarkdownLayout.astro'
 ---
 
-_Last updated_: January 06, 2023
-
 Please read these terms and conditions carefully before using Our Service.
 
-## Interpretation and Definitions
```

Comment:
I actually think these are reasonable terms we could probably keep them.

Reply:
That being said, could mention that askinternally is a internal self hosted variant of askremotely so therefore the terms of service is more of whatever the self hosting platform / entity would be.